app/serializers/api/public/v1/serializable_location.rb

```diff
@@ -0,0 +1,43 @@
+module API
+  module Public
+    module V1
+      class SerializableLocation < JSONAPI::Serializable::Resource
+        type "locations"
+
+        belongs_to :recruitment_cycle
+        belongs_to :provider
+
+        belongs_to :course do
+          data { @course }
+        end
```

Comment:
why do you need `data`?
```suggestion
        belongs_to :course
```
will the above alone not work?

Reply:
No. A site does not have a `belong_to` association to a course. Data allows you to explicitly control what data to declare under the attribute.